### PR TITLE
Added Azure AD MFA Auth with O365

### PR DESCRIPTION
**Pull request by Malind19**

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/identity-protection/hello-for-business/hello-hybrid-key-trust-prereqs.md [hunk 1]
```diff
@@ -97,7 +97,7 @@ You can deploy Windows Hello for Business key trust in non-federated and federat
 ## Multifactor Authentication ##
 Windows Hello for Business is a strong, two-factor credential the helps organizations reduce their dependency on passwords.  The provisioning process lets a user enroll in Windows Hello for Business using their user name and password as one factor, but needs a second factor of authentication.
 
-Hybrid Windows Hello for Business deployments can use Azure’s Multi-factor Authentication service or they can use multi-factor authentication provides by Windows Server 2012 R2 or later Active Directory Federation Services, which includes an adapter model that enables third parties to integrate their multi-factor authentication into AD FS.
+Hybrid Windows Hello for Business deployments can use Azure’s Multi-factor Authentication service or they can use multi-factor authentication provides by Windows Server 2012 R2 or later Active Directory Federation Services, which includes an adapter model that enables third parties to integrate their multi-factor authentication into AD FS. The Multi-factor authentication enabled in Office 365 license is sufficient for direct Multi-factor Authentication against Azure AD.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-key-trust-prereqs.md]:
```suggestion
Hybrid Windows Hello for Business deployments can use Azure’s Multi-factor Authentication service or they can use multi-factor authentication provided by Windows Server 2012 R2 or later Active Directory Federation Services, which include an adapter model that enables third parties to integrate their multi-factor authentication into AD FS. The Multi-factor authentication enabled in Office 365 license is sufficient for direct Multi-factor Authentication against Azure AD.
```

Comment (nenonix) [reply windows/security/identity-protection/hello-for-business/hello-hybrid-key-trust-prereqs.md]: I agree

**Comment by mypil** [comment]
@Malind19 - please merge the commit on the copy/edit review made by JohanFreelancer9 or make the necessary changes on the PR following the suggestions he made. Thank you.

**Comment by arourke** [comment]
This change also needs to be clarified on the [this page](https://docs.microsoft.com/en-us/windows/security/identity-protection/hello-for-business/hello-hybrid-key-new-install#azure-multi-factor-authentication-mfa-cloud), it should be added to the list of MFA licensing options

**Comment by Malind19** [comment]
@mypil Done.

**Comment by mypil** [comment]
@DaniHalfin @mapalko - Can you please assist to check, approve and merge the content for this PR based on issue #3296?

Thank you.

**Comment by mypil** [comment]
@DaniHalfin @mapalko - I would like to follow up on this. Can you please assist to check, approve and merge the content for this PR based on issue #3296?

Thank you.

**Comment by mypil** [comment]
@AndreaBarr - would you be able to merge the content of this PR? @mapalko has approved the changes. Thank you.

**Comment by mypil** [comment]
@AndreaBarr @DaniHalfin @kenwith - would you be able to merge the content of this PR? @mapalko has approved the changes. Thank you.

**Comment by mypil** [comment]
@Justinha @lizap @mjcaparas @DaniHalfin @AndreaBarr -  would you be able to merge the content of this PR? @mapalko has approved the changes. Thank you.

**Pull request merged**